webapp/src/monaco.tsx

```diff
@@ -728,6 +728,9 @@ export class Editor extends toolboxeditor.ToolboxEditor {
             });
 
             if (monacoToolboxDiv) monacoToolboxDiv.style.height = `100%`;
+
+            // ensures all lines are displayed
+            this.editor.focus();
```

Comment:
Does this mean that the monaco editor will steal focus when the user resizes the window?

Reply:
Great question, we don't want to do that. Hristo, we might need to dig into .focus() to see what it's actually doing that fixes the sizing issue and maybe only call a subset of focus()'s behavior.

Reply:
Also it's worth testing this with tab navigation. If you're tabbing around the page and this triggers does  the user lose their tab location?

Reply:
Ok I'll look into this more to see if there is another way

Reply:
```suggestion
            this.editor.layout();
```

@hristoramirez could you check if this fixes the problem? at a glance it looks like the issue is that the dimensions and offset are hardcoded on each of the divs (e.g. `div.monaco-scrollable-element` that's a parent of the individual lines), and that gets re-evaluated on focus: https://github.com/microsoft/monaco-editor/blob/master/monaco.d.ts#L2133

Reply:
@jwunderl I tried it, but it gets stuck in a loop while loading

Reply:
@jwunderl so far it looks like it fixes the problem

Reply:
Might be worth checking that this doesn't slow down the editor resize speed